src/components/SettlementButton/index.tsx

```diff
@@ -329,7 +252,7 @@ function SettlementButton({
 
 SettlementButton.displayName = 'SettlementButton';
 
-export default withOnyx<SettlementButtonProps, SettlementButtonOnyxProps>({
+export default withOnyx<SettlementButtonWithOnyxProps, SettlementButtonOnyxProps>({
```

Comment:
seems like this would be an easy one to simplify by refactoring `withOnyx` to `useOnyx`. We just merged a PR that makes this a new requirement: https://github.com/Expensify/App/pull/49125